virtual_rainforest/data_variables.toml

```diff
@@ -0,0 +1,385 @@
+[[variable]]
+name = "air_temperature"
+description = "air temperature profile"
+unit = "C"
+source = "abiotic"
+used_by = [ "abiotic", "plants", "animals"]
+
+[[variable]]
+name = "air_temperature_ref"
+description = "air temperature at reference height (2m)"
+unit = "C"
+source = "external"
+used_by = [ "abiotic"]
+
+[[variable]]
+name = "albedo_shortwave"
+description = "shortwave light albedo"
+unit = "-"
+source = "external"
+used_by = [ "abiotic"]
+
+[[variable]]
+name = "albedo_vis"
+description = "visible light albedo"
+unit = "-"
+source = "external"
+used_by = [ "abiotic"]
+
+[[variable]]
+name = "animal_respiration"
+description = "animal respiration aggregated over all functional types"
+unit = "ppm"
+source = "animal"
+used_by = [ "abiotic", "animals"]
+
+[[variable]]
+name = "atmospheric_co2_ref"
+description = "CO2 concentration at reference height (above canopy)"
+unit = "ppm"
+source = "external"
+used_by = [ "abiotic"]
+
+[[variable]]
+name = "atmospheric_pressure"
+description = "atmospheric pressure profile"
+unit = "kPa"
+source = ["abiotic"] 
```

Comment:
There is an inconsistent structure for `source`: mostly it is a string, but sometimes it is a list of string(s). Given @jacobcook1995 's comment about "external + soil", they should probably all be lists?